src/toil_scripts/adam_uberscript/adam_uberscript.py

```diff
@@ -119,8 +118,13 @@ def launch_pipeline(params):
                                '-o', 'UserKnownHostsFile=/dev/null',
                                '-o', 'StrictHostKeyChecking=no',
                                'screen', '-dmS', params.cluster_name])
-        # Run command on screen session
-        
+        # Run command on screen session  
+
+        if params.reference_genome == 'GRCh38':
+            from toil_scripts.adam_uberscript.input_files import GRCh38_inputs as inputs
+        if params.reference_genome == 'hg19':
+            from toil_scripts.adam_uberscript.input_files import hg19_inputs as inputs       
```

Comment:
> Yes, validate early and raise RuntimeException to alert user but also assert here. If someone changes the validation code and forgets to change this code here, they will have introduced a bug which would be caught by the assert.

Just wanted to point out that the flip side of this is that if someone adds a new reference build, then they need to modify validation code in two places. My preference is to validate the inputs in a single place, and to validate them as early as is possible.

> And -1 on informational message, the stack trace will be informative enough.

Are you actually _-1_ on this? It doesn't make sense to me to require the end user to parse out a stack trace when you can raise an informative error message instead.